### PR TITLE
Request code review for Tic tac toe project

### PR DESCRIPTION
**Pull request by majovanilla**
We are requesting a review for the Tic Tac Toe Project.

We included the required features and specifications on both the Odin Project and Microverse.

We would like to ask for your advice in two linter errors that we have:
One is about using destructuring to create an object (we couldn't figure out how to use it in this case)
The other one is about the use of location (we didn't understand how to solve it)

Thanks for your help!

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+      gameBoard.cell2.textContent = gameBoardArr[1];
+      gameBoard.cell3.textContent = gameBoardArr[2];
+      gameBoard.cell4.textContent = gameBoardArr[3];
+      gameBoard.cell5.textContent = gameBoardArr[4];
+      gameBoard.cell6.textContent = gameBoardArr[5];
+      gameBoard.cell7.textContent = gameBoardArr[6];
+      gameBoard.cell8.textContent = gameBoardArr[7];
+      gameBoard.cell9.textContent = gameBoardArr[8];
+    },
+
```
Comment (shloch) [inline main.js]: You can do this :
```
const [a, b, c, d, e, f, g, h, i] = [...gameBoardArr];
      gameBoard.cell1.textContent = a;
      gameBoard.cell2.textContent = b;
      gameBoard.cell3.textContent = c;
      gameBoard.cell4.textContent = d;
      gameBoard.cell5.textContent = e;
      gameBoard.cell6.textContent = f;
      gameBoard.cell7.textContent = g;
      gameBoard.cell8.textContent = h;
      gameBoard.cell9.textContent = i;
```

File: main.js [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+      UICtrl.displayScore(player1.score, player2.score);
+      newRound();
+    } else if (counter === 9 && checkDraw() === true) {
+      UICtrl.displayResult(true);
+    }
+  };
+
+  const resetGame = () => {
+    location.reload();
+  };
```
Comment (shloch) [inline main.js]: add this linter config comment (do this): 
```
  /* eslint no-restricted-globals: ["error", "event"] */
  const resetGame = () => {
    location.reload();
  };
```
documentation about this linter error here --> [click this](https://eslint.org/docs/rules/no-restricted-globals)

File: main.js (hunk 1 above)
Comment (majovanilla) [reply main.js]: Thanks for the tip!! =)

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```
Comment (bolah2009) [inline .eslintrc.json]: Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/javascript).

> Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

Comment (majovanilla) [reply .eslintrc.json]: Thanks!! I don't know why that happens. We had them ok before. But it's back the way it should be.

**Pull request merged**